src/core/FxMixer.cpp

```diff
@@ -741,6 +742,8 @@ void FxMixer::saveSettings( QDomDocument & _doc, QDomElement & _this )
 		ch->m_soloModel.saveSettings( _doc, fxch, "soloed" );
 		fxch.setAttribute( "num", i );
 		fxch.setAttribute( "name", ch->m_name );
+		fxch.setAttribute( "hascolor", ch->m_hasColor );
+		fxch.setAttribute( "mixercolor", ch->m_color.rgb() );
```

Comment:
> Also, how about saving `m_color.name()` instead?

Is this necessary? Don't exactly get why this is needed because saving rgb seems more straightforward and more flexible. Besides the palette colors aren't named ones anyways...

Reply:
`QColor::name()`, despite its name, actually returns the colour in the format `#RRGGBB` - see the documentation I linked above. `QColor` can also be constructed from such a string, so it doesn't complicate saving or loading at all.

Reply:
Ah, I misinterpreted that as a function that gives something like `red` or `green`. Tried it out, seems more readable and straightforward, so am using it now.